integration_test/third_party_apps_data/agent/ops-agent/windows/install

```diff
@@ -0,0 +1,8 @@
+$ProgressPreference = 'SilentlyContinue'
+
+# These are the steps from our public documentation:
+# https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/installation#joint-install
+# It's important that these steps exactly match the ones we tell our
+# customers to run.
+(New-Object Net.WebClient).DownloadFile("https://dl.google.com/cloudagents/add-google-cloud-ops-agent-repo.ps1", "${env:UserProfile}\add-google-cloud-ops-agent-repo.ps1")
+Invoke-Expression "${env:UserProfile}\add-google-cloud-ops-agent-repo.ps1 -AlsoInstall"
```

Comment:
[Optional] This will inherit `REPO_SUFFIX` from the environment, which may be a bit counter-intuitive. Want to add a comment?

Reply:
done.